.github/workflows/formatter.yml

```diff
@@ -0,0 +1,11 @@
+on: push
```

Comment:
Make sure we aren't committing this file addition after rebase.

Reply:
Deleted these files